pkg/kubectl/cmd/cp/cp.go

```diff
@@ -454,36 +460,9 @@ func (o *CopyOptions) untarAll(reader io.Reader, destDir, prefix string) error {
 			continue
 		}
 
-		// We need to ensure that the destination file is always within boundries
-		// of the destination directory. This prevents any kind of path traversal
-		// from within tar archive.
-		evaledPath, err := filepath.EvalSymlinks(baseName)
-		if err != nil {
-			return err
-		}
-		// For scrutiny we verify both the actual destination as well as we follow
-		// all the links that might lead outside of the destination directory.
-		if !isDestRelative(destDir, filepath.Join(evaledPath, filepath.Base(destFileName))) {
-			fmt.Fprintf(o.IOStreams.ErrOut, "warning: file %q is outside target destination, skipping\n", destFileName)
-			continue
-		}
-
 		if mode&os.ModeSymlink != 0 {
-			linkname := header.Linkname
-			// We need to ensure that the link destination is always within boundries
-			// of the destination directory. This prevents any kind of path traversal
-			// from within tar archive.
-			linkTarget := linkname
-			if !filepath.IsAbs(linkname) {
-				linkTarget = filepath.Join(evaledPath, linkname)
-			}
-			if !isDestRelative(destDir, linkTarget) {
-				fmt.Fprintf(o.IOStreams.ErrOut, "warning: link %q is pointing to %q which is outside target destination, skipping\n", destFileName, header.Linkname)
-				continue
-			}
-			if err := os.Symlink(linkname, destFileName); err != nil {
-				return err
-			}
+			fmt.Fprintf(o.IOStreams.ErrOut, "warning: file %q is a symlink, skipping (consider using %s exec for symlink support)\n", destFileName, o.cmd.Parent().Name())
```

Comment:
nit: this is verbose for an error that might be printed a bunch of times. Maybe simplify it to something like:
```suggestion
			fmt.Fprintf(o.IOStreams.ErrOut, "skipping symlink: %q -> %q\n", destFileName, linkTarget)
```

Reply:
I'd tend to agree. the pointer to exec can be printed just once